.gitignore

```diff
@@ -0,0 +1 @@
+*.md
```

Comment:
- [ ]  You should also include the `test.md` file and `node_modules`  in the `.gitignore` file as it's a requirement in the project